components/Storyblok/SbStoryPage/SbStoryFullView.tsx

```diff
@@ -118,19 +118,19 @@ export const SbStoryFullView = (props: SbStoryFullViewProps) => {
                     <div className="ood-story__metadata su-pb-5">
                       {props.blok.author && (
                         <>
-                          <p className="ood-story__metadata-title su-bold su-uppercase">
+                          <Paragraph weight="bold" uppercase className="ood-story__metadata-title">
```

Comment:
`span` or `heading` probably more appropriate for these but we can deal with them later too 😄